account_fiscal_year_reopen/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Use latest README template with 2 lines in the title, target... https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst